### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by tczaude**
I change branch and last branch is lose
Now All commits it there

### CODE REVIEW

File: install/lib/class.FroxlorInstall.php [hunk 1]
```diff
@@ -66,7 +66,12 @@ class FroxlorInstall {
 	private $_languages = array(
 			'german' => 'Deutsch',
 			'english' => 'English',
-			'french' => 'Français'
+			'french' => 'Français',
+<<<<<<< af10fe69bd5116e0fcafd4bbe8e3cd14bf471075
```
Comment (d00p) [inline install/lib/class.FroxlorInstall.php]: Why commit conflicts?

File: install/froxlor.sql [hunk 2]
```diff
@@ -655,7 +655,8 @@ INSERT INTO `panel_languages` (`id`, `language`, `iso`, `file`) VALUES
     (4, 'Portugu&ecirc;s', 'pt', 'lng/portugues.lng.php'),
     (5, 'Italiano', 'it', 'lng/italian.lng.php'),
     (6, 'Nederlands', 'nl', 'lng/dutch.lng.php'),
-    (7, 'Svenska', 'sv', 'lng/swedish.lng.php');
+    (7, 'Svenska', 'sv', 'lng/swedish.lng.php'),
+    (8, 'Polish', 'pl', 'lng/polish.lng.php');
```
Comment (d00p) [inline install/froxlor.sql]: again, same as ion FroxlorInstall, don't use the english name of the language but the polish word for "polish"

Comment (tczaude) [inline install/froxlor.sql]: Hi i remember this sorry 
i check this I lost some little lines
But We make this that (For backup)
Give me 1 day i should give final commit

File: install/lib/class.FroxlorInstall.php (hunk 1 above)
Comment (tczaude) [inline install/lib/class.FroxlorInstall.php]: My local repo make this :/

**Pull request closed without merging**